### PR TITLE
API-13230: Release notes for Claims API

### PR DESCRIPTION
**Pull request by hinzed1127**
### Description

Release notes for the changes made by https://github.com/department-of-veterans-affairs/vets-api/pull/9112
### Process

~- [ ] Tests added or updated for change~
- [X] Docs updated
  ~- [ ] If you have made structural changes to the site, the [Developer Portal Content Types](https://community.max.gov/x/qUwOg) docs have been updated~
  ~- [ ] If you have made/confirmed any decisions about user interactions and accessibility, the [Accessible Component Design](https://community.max.gov/x/xS8mgg) docs have been updated~
~- [ ] Accessibility concerns have been considered and addressed~
  
  ~- [ ] If you've made any UI changes, you've run an axe check using the axe browser extension that reported no new issues~

### CODE REVIEW

**Review by gileswells — DISMISSED** [review]
LGTM if UX approves or has approved feel free to merge

File: src/content/apiDocs/benefits/claimsReleaseNotes.mdx [hunk 1]
```diff
@@ -1,10 +1,20 @@
+### February 15, 2022
+
+- Updated the v1 validation endpoints to use the same validation as their respective submission endpoints. The following endpoints validate in the same way now ([#9112](https://github.com/department-of-veterans-affairs/vets-api/pull/9112)):
```
Comment (joeabraun) [inline src/content/apiDocs/benefits/claimsReleaseNotes.mdx:3]:
```suggestion
- Updated the v1 validation endpoints to validate in the same way as their respective submission endpoints. These following endpoints are:
```

File: src/content/apiDocs/benefits/claimsReleaseNotes.mdx [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
+  - POST "forms/526" and "forms/526/validate"
+  - POST "forms/0966" and "forms/0966/validate"
+  - POST "forms/2122" and "forms/2122/validate"
+
```
Comment (joeabraun) [inline src/content/apiDocs/benefits/claimsReleaseNotes.mdx:7]: [#9112](https://github.com/department-of-veterans-affairs/vets-api/pull/9112)

**Review by joeabraun — CHANGES_REQUESTED** [review]
Reworded the two sentences and moved the number until after the listed endpoints.

File: src/content/apiDocs/benefits/claimsReleaseNotes.mdx (hunk 1 above)
Comment (hinzed1127) [reply src/content/apiDocs/benefits/claimsReleaseNotes.mdx]: @joeabraun "These following" is redundant, so I just changed to "These" and removed "following" entirely. Rest of the edits were made as commented above.

Comment (joeabraun) [reply src/content/apiDocs/benefits/claimsReleaseNotes.mdx]: @hinzed1127 okay, great. Looks good to me.

**Review by SlakrHakr — APPROVED** [review]

**Pull request merged**